apps/src/StudioApp.js

```diff
@@ -532,6 +503,49 @@ StudioApp.prototype.init = function(config) {
 };
 
 /**
+ * Sets html for prompts below playspace, anigif, and shows instructions dialog
+ */
+StudioApp.prototype.configureAndShowInstructions_ = function (config) {
```

Comment:
By type name, do you mean something we put in our @param tag?

It seems like whatever name we used also did something to make it clear that this is the same as appOptions.

What about something like AppOptionConfiguration (or just AppOptionConfig)?